workshops/forms.py

```diff
@@ -340,37 +341,34 @@ class EventForm(forms.ModelForm):
     admin_fee = forms.DecimalField(min_value=0, decimal_places=2,
                                    required=False, widget=TextInput)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.fields['slug'].widget.attrs['placeholder'] = 'YYYY-MM-DD-location'
-        self.fields['start'].widget.attrs['placeholder'] = 'YYYY-MM-DD'
-        self.fields['end'].widget.attrs['placeholder'] = 'YYYY-MM-DD'
-
-        self.helper = BootstrapHelper(self)
-
-        idx_start = self.helper['country'].slice[0][0][0]
-        idx_end = self.helper['longitude'].slice[0][0][0]
-        # wrap all venue fields within <div class='panel-body'>
-        self.helper[idx_start:idx_end + 1] \
-            .wrap_together(Div, css_class='panel-body')
-        # wrap <div class='panel-body'> within <div class='panel panel-…'>
-        self.helper[idx_start].wrap_together(Div,
-                                             css_class='panel panel-default')
-        # add <div class='panel-heading'>Loc. details</div> inside "div.panel"
-        self.helper.layout[idx_start].insert(0, Div(HTML('Location details'),
-                                                    css_class='panel-heading'))
-
-        id_learners_pre = self.helper['learners_pre'].slice[0][0][0]
-        id_learners_longterm = self.helper['learners_longterm'].slice[0][0][0]
-        # wrap all survey fields within <div class='panel-body'>
-        self.helper[id_learners_pre:id_learners_longterm + 1] \
-            .wrap_together(Div, css_class='panel-body')
-        # wrap <div class='panel-body'> within <div class='panel panel-…'>
-        self.helper[id_learners_pre].wrap_together(
-            Div, css_class='panel panel-default')
-        # add <div class='panel-heading'>Venue details</div> inside "div.panel"
-        self.helper.layout[id_learners_pre].insert(
-            0, Div(HTML('Survey results'), css_class='panel-heading'))
+    helper = BootstrapHelper()
+    helper.layout = Layout(
+        Field('slug', placeholder='YYYY-MM-DD-location'),
+        'completed',
+        Field('start', placeholder='YYYY-MM-DD'),
+        Field('end', placeholder='YYYY-MM-DD'),
+        'host',
+        'administrator',
+        'assigned_to',
+        'tags',
+        'url',
+        'language',
+        'reg_key',
+        'admin_fee',
+        'invoice_status',
+        'attendance',
+        'contact',
+        'notes',
+        Accordion(
+            AccordionGroup('Location details',
```

Comment:
So previously we had some implementation that's now in the `Accordion`? The output look is the same -- which means less code maintenance for us (big win)!